app/views/idv/gpo_verify/index.html.erb

```diff
@@ -11,7 +11,10 @@
 
 <% title t('titles.verify_profile') %>
 
-<%= render PageHeadingComponent.new.with_content(t('forms.verify_profile.title')) %>
+<%= render PageHeadingComponent.new.with_content(t('forms.verify_profile.welcome_back')) %>
+<%= sanitize(t('forms.verify_profile.welcome_back_description')) %>
+<hr/>
```

Comment:
Looks like the margin in the design is a bit bigger than in the screenshot.

```suggestion
<hr class="margin-y-4" />
```

Reply:
The details my brain will never see 🙈

Reply:
> The details my brain will never see 🙈

I think the reason I thought to consider it is that we don't have much in terms of default styling for `<hr />` 😄 I think it could be good to add some.